.github/pipelines.sh

```diff
@@ -2,11 +2,18 @@
 set -e
 YQ="${YQ:-yq}"
 
+VERSION=$($YQ -V |rev|  cut -d " " -f 1 |rev| cut -d "." -f 1)
+if [[ "$VERSION" == "3" ]]; then
+    echo "yq version 3 detected, only version 4 is supported"
```

Comment:
:heart:

Reply:
Im not sure if leaving that in place and add it everywhere or just drop it altogether